obs-browser-plugin.cpp

```diff
@@ -390,7 +392,10 @@ void RegisterBrowserSource()
 	};
 	info.show = [] (void *data)
 	{
-		static_cast<BrowserSource *>(data)->SetShowing(true);
+		BrowserSource *bs = static_cast<BrowserSource *>(data);
+		if (bs->restart_on_visible)
+			bs->Refresh();
```

Comment:
The option you're likely looking for is actually `bs->cefBrowser->ReloadIgnoreCache();` if you're going off the #164 request.